docker-compose.yml

```
@@ -26,7 +26,7 @@ services:
 
   # Collector
   otelcol:
-    image: otel/opentelemetry-collector:0.56.0
+    image: otel/opentelemetry-collector-contrib:0.52.0
```

Can we have collector-contrib 0.56.0?
```suggestion
    image: otel/opentelemetry-collector-contrib:0.56.0
```